### PR TITLE
feat: using fallback function when matched loans are less than 3 inst…

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Using fallback function when matched loans are less than 3 instead of less than 1 in lending home page

### CODE REVIEW

File: src/pages/LoanFinding/LoanFinding.vue [hunk 1]
```diff
@@ -198,7 +198,7 @@ export default {
 		async getSecondCategoryData() {
 			this.secondCategoryLoans = await this.getMatchedLoans();
 			this.matchedLoansTotal = this.secondCategoryLoans.length;
-			if (this.matchedLoansTotal === 0) this.secondCategoryLoans = await this.getExpiringSoonAlmostFundedCombo(); // eslint-disable-line max-len
+			if (this.matchedLoansTotal < 3) this.secondCategoryLoans = await this.getExpiringSoonAlmostFundedCombo(); // eslint-disable-line max-len
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:201]: We actually want to combine the matched with the fallback. So the first 1-2 loans are matched, and the rest are the fallback. Here we are replacing the matched loans results with the fallback.

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:201]: oh, I see the note 🤦‍♂️, ty

File: src/pages/LoanFinding/LoanFinding.vue [hunk 2]
```diff
@@ -198,7 +199,11 @@ export default {
 		async getSecondCategoryData() {
 			this.secondCategoryLoans = await this.getMatchedLoans();
 			this.matchedLoansTotal = this.secondCategoryLoans.length;
-			if (this.matchedLoansTotal === 0) this.secondCategoryLoans = await this.getExpiringSoonAlmostFundedCombo(); // eslint-disable-line max-len
+			if (this.matchedLoansTotal < 3) {
+				const fallbackLoans = await this.getExpiringSoonAlmostFundedCombo();
+				this.secondCategoryLoans = [...this.secondCategoryLoans, ...fallbackLoans].slice(0, 9);
+			}
+			this.secondCategoryActive = true;
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:206]: I don't think this new flag is needed, since the source of the loans is still `secondCategoryLoans`

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:206]: yeah, but if the first assignment is less than 9, this is rendered with them and not updated later... other way is to have `secondCategoryLoans` as a computed prop and remove the flag

Comment (dyersituations) [reply src/pages/LoanFinding/LoanFinding.vue:206]: Use a local variable in that method, then assign at the end.

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:206]: Good!

**Review by dyersituations — APPROVED** [review]

**Pull request merged**